api/envoy/api/v2/rds.proto

```diff
@@ -102,3 +130,9 @@ message RouteConfiguration {
   // using CDS with a static route table).
   google.protobuf.BoolValue validate_clusters = 7;
 }
+
+message Vhds {
```

Comment:
This message should be `// [#not-implemented-hide:]`; it's visible at the CI generated docs https://192807-65214191-gh.circle-artifacts.com/0/source/generated/docs/api-v2/api/v2/rds.proto.html

Reply:
fixed